src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java

```diff
@@ -105,6 +109,20 @@ public InputStream preview(Integer mediaId) {
         return mediaBinaryData.toStream();
     }
 
+    @Override
+    public Flux<ResponseEntity<byte[]>> getAudioPreviewFlux(Integer mediaId, String range) {
```

Comment:
what subject is this publishing to, as it will need to match what the front-end is looking out for

Reply:
I have changed it to SSE events as the front end is looking at the event with name "heartbeat" to determine.